### ISSUE TITLE
Error: open : The system cannot find the file specified.

### ISSUE THREAD

**Issue by SzabKel** [digitalocean/doctl#1281]
**Describe the Issue:**
[Following the guide here](https://docs.digitalocean.com/reference/doctl/how-to/install/), I downloaded doctl.exe, put it on the path. Running `doctl auth init` does not work however (tried it with powershell, CMD, with and without administrator rights).

It asks for the access token, then throws an error:
![image](https://user-images.githubusercontent.com/3736434/195769195-51fbc5ad-353e-4aea-9776-0dfbba739634.png)

`Error: open : The system cannot find the file specified.`

**Environment:**

 - doctl version: doctl-1.83.0-windows-amd64
 - OS: Windows 10 21H2
 - Installation method: manually downloaded the exe

**Additional Details:**

The `%APPDATA%\doctl\config.yaml` file exists and if I try to auth init with a `--name` it will also save the access token to the config file. However running `auth switch --context <name>` throws the save error. Running `doctl account get` does not work.

Manually specifying the config path does not work as well (the file exists however):
![image](https://user-images.githubusercontent.com/3736434/195771714-bfb08317-631b-43a6-9243-f3df7d04cb91.png)

**Comment by andrewsomething** [comment]
Hi @SzabKel,

Thanks for flagging this for us. I haven't been able to reproduce the problem yet myself.

`%APPDATA%` is a variable that should be expanded. What does `$env:APPDATA` output for you? Are you able to write to that directory?  You should be able to check permission on the file itself with something like: 

    ​(Get-ACL -Path "\path\to\config.yaml").Access | Format-Table IdentityReference,FileSystemRights,AccessControlType,IsInherited,InheritanceFlags -AutoSize

**Comment by emersonp** [comment]
Running into the same error as OP.

My `$env:APPDATA` outputs `C:\Users\parke\AppData\Roaming\`, and within I can find a `doctl\config.yaml`.

The `(Get-ACL`... command throws an error (maybe I'm bad at PowerShell?)
```(Get-ACL -Path "C:\Users\parke\AppData\Roaming\doctl\config.yaml").Access | Format-Table IdentityReference,FileSystemRights,AccessControlType,IsInherited,InheritanceFlags -AutoSize```

```​ 
: The term '​' is not recognized as the name of a cmdlet, function, script file, or operable program. Check the
spelling of the name, or if a path was included, verify that the path is correct and try again.
At line:1 char:1
+ ​(Get-ACL -Path "C:\Users\parke\AppData\Roaming\doctl\config.yaml").A ...
+ ~
    + CategoryInfo          : ObjectNotFound: (​:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException
```

Doubling the backslashes doesn't help.

> What does `doctl --help` output as the default for the `--config` flag?

`Specify a custom config file (default "C:\\Users\\parke\\AppData\\Roaming\\doctl\\config.yaml")`

**Comment by SzabKel** [comment]
@andrewsomething 

> `%APPDATA%` is a variable that should be expanded. What does `$env:APPDATA` output for you? Are you able to write to that directory? 
![image](https://user-images.githubusercontent.com/3736434/196094508-21436a7f-a581-4e67-b87f-978f3b6a7886.png)

I can write to that directory yes. 

![image](https://user-images.githubusercontent.com/3736434/196094543-1a6ad20e-6b4a-44c7-9d52-788a5c39c5ef.png)

> You should be able to check permission on the file itself with something like:

![image](https://user-images.githubusercontent.com/3736434/196094887-737b063c-3c43-47df-a0c5-9078b3f51239.png)

> What does doctl --help output as the default for the --config flag?

![image](https://user-images.githubusercontent.com/3736434/196095236-edae0be9-9ddd-45da-9869-c6980a8323c8.png)

**Comment by SzabKel** [comment]
Manually passing in the config seems to work.
![image](https://user-images.githubusercontent.com/3736434/196375327-dc961ff2-128c-48da-8cd6-f8588f53b58a.png)

`doctl account get` works after this.

**Comment by andrewsomething** [comment]
I believe we have tracked down the root of this problem. It should be fixed in the [v1.84.0 release](https://github.com/digitalocean/doctl/releases/tag/v1.84.0) via https://github.com/digitalocean/doctl/pull/1286. Unfortunately, the issue was with the config file itself. So if you have been experiencing this problem, you will need to either delete your existing config file or edit it to remove the stanza that looks like:

```
config:
  set:
    dev-config: ""
```

Please let us know if this has resolved the issue for you.

**Comment by SzabKel** [comment]
@andrewsomething I can confirm, the new version fixed the issue.
